lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -53,6 +55,8 @@ IDENTITY_REUSE_PAGE_TEST:
   events:
     next:
       targetState: NEW_DETAILS_PAGE
+    update-details:
+      targetState: UPDATE_DETAILS_PAGE
```

Comment:
Probably not something to include in this PR, but I think this state and the new details/confirm details pages below can be deleted now - they're superseded by the COI journeys.

(You can check with @RogVanGDS next week if you're unsure)